.openpublishing.redirection.json

```diff
@@ -1801,7 +1809,7 @@
         },
         {
             "source_path": "docs/framework/migration-guide/runtime-changes-in-the-net-framework-4-6-2.md",
-            "redirect_url": "/dotnet/framework/migration-guide/runtime/index"   
+            "redirect_url": "/dotnet/framework/migration-guide/runtime/index"
```

Comment:
you just removed this one, should you retarget?

Reply:
Good catch.